src/test/java/com/artipie/rpm/RpmITCase.java

```diff
@@ -280,4 +278,14 @@ private static String readGzFile(final File file) throws IOException {
             return res.toString();
         }
     }
+
+    /**
+     * Inner subclass to instantiate yum-utils container.
+     * @since 0.3.3
+     */
+    private static class YumUtilsContainer extends GenericContainer<YumUtilsContainer> {
```

Comment:
@brastak lets make this class public to make it reusable

Reply:
@Vatavuk I think this docker image is too specialized to become reusable. This is base centos image with 2 additional packages. It is reasonable idea to create the set of docker container and use them for testing different adapters, but I don't think that this concrete container will be useful anywhere except RPM adapter. 
I created nested class here just because of current testcontainers limitation. In the other cases I would prefer to localize docker image information in the container variable declaration. Something like this:
`new GenericContainer("yegor256/yum-utils").with...`

Reply:
@brastak ok, seems reasonable